### PR TITLE
Update connect-to-gdn.md

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
Updating connect to GDN to match the code sample from SDK.md, also mention the use of apikey and JWT

### CODE REVIEW

File: docs/sdks/connect-to-gdn.md [hunk 1]
```diff
@@ -13,7 +13,40 @@ The first step in using GDN is to establish a connection to a local region. When
 
 ```js
 const jsc8 = require("jsc8");
-const client = new jsc8({url: "https://gdn.paas.macrometa.io", apiKey: "xxxx", fabricName: '_system'});
+
+// Email and Password to Authenticate client instance
```
Comment (diana-macrometa) [inline docs/sdks/connect-to-gdn.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/sdks/connect-to-gdn.md [hunk 2]
```diff
@@ -13,7 +13,40 @@ The first step in using GDN is to establish a connection to a local region. When
...
+const email = "nemo@nautilus.com";
+const password = "xxxxxx";
+const fabric = "_system";
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+// Or create an authenticated instance with a JWT token.
```
Comment (diana-macrometa) [inline docs/sdks/connect-to-gdn.md:26]:
```suggestion
// Choose one of the following authentication methods and remove the commenting.

// Create an authenticated instance with a JWT token.
```

File: docs/sdks/connect-to-gdn.md [hunk 3]
```diff
@@ -13,7 +13,40 @@ The first step in using GDN is to establish a connection to a local region. When
...
+const clientUsingJwt = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  token: "XXXX",
+  fabricName: fabric
+});
+// Or create an authenticated instance with a API key.
```
Comment (diana-macrometa) [inline docs/sdks/connect-to-gdn.md:32]:
```suggestion
// Create an authenticated instance with a API key.
```

**Review by diana-macrometa — APPROVED** [review]
A few suggestions.

File: docs/sdks/connect-to-gdn.md [hunk 4]
```diff
@@ -13,7 +13,40 @@ The first step in using GDN is to establish a connection to a local region. When
...
```
Comment (jamesm-macrometa) [inline docs/sdks/connect-to-gdn.md]:
```suggestion
// Or create an authenticated instance with an API key.
```

**Review by jamesm-macrometa — APPROVED** [review]

**Pull request merged**